### ISSUE TITLE
The leakz module has a bug when working with Python 2

### ISSUE THREAD

**Issue by Jolly979** [thehappydinoa/leakz#1]
```
root@GithackTools618:~/clearram# python
Python 2.7.15+ (default, Aug 31 2018, 11:56:52) 
[GCC 8.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import leakz
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/leakz/__init__.py", line 1, in <module>
    from .api import *
  File "/usr/local/lib/python2.7/dist-packages/leakz/api.py", line 6
    def api_request(path: str, base_url: str = "https://lea.kz") -> dict:
                        ^
SyntaxError: invalid syntax
>>>'
```

**Comment by thehappydinoa** [comment]
Ah ok, I totally forgot Python 2 doesn't support type casting.

**Comment by Jolly979** [comment]
Leaked? should still only supports Python3

**Comment by thehappydinoa** [comment]
Fixed in c475cb1
